pkg/arvo/sys/vane/gall.hoon

```diff
@@ -1753,6 +1765,16 @@
       ==
     [~ ~ noun+!>((~(has by yokes.state) dap))]
   ::
+  ?:  ?&  =(%d care)
+          =(~ path)
+          =([%$ %da now] coin)
+          =(our ship)
+      ==
+    =/  yok=(unit yoke)  (~(get by yokes.state) dap)
+    ?~  yok
+      [~ ~]
+    [~ ~ desk+!>(q.beak.u.yok)]
```

Comment:
Worth noting that clay implements the `%d` care as "set of all desks". If gall is going to continue using the desk field for app names, we might want to have `%gd` produce a set of all running apps instead.

At this point clay also implements a handful of less-generic cares though, certainly no other vane is going to implement a care for mark conversions. Wondering how to best delineate between generic and special-purpose cares, and whether the latter should exist in the first place...